docs/src/configuration/routes/_index.md

```diff
@@ -276,14 +276,13 @@ The full list of generated route information is often much larger than what's sp
 For example, by default all HTTPS routes (and all uses of `{all}`) are duplicated to create HTTP redirect routes.
 As a general rule, you should keep to your defined routes under 100.
 
-If your `routes.yaml` file is rejected for being too big,
-the best alternative is to move redirect routes to the application rather than relying on the router.
-You can also collapse multiple routes into a [regular expression-based redirect](./redirects.md#partial-redirects).
-
-Let's Encrypt has a limit of 100 hostnames per certificate and
-each of your environments can have 1 Let's Encrypt certificate.
-If you define both a `{default}` and `www.{default}` route for each domain you use, you have a limit of 50 hostnames.
-Adding more than that results in a warning on deploy and no new TLS certificates are issued.
+Should your `routes.yaml` file be rejected due to an excessive size, either:
+- move any redirect routes to the application,
```

Comment:
```suggestion
- move redirect routes to the application,
```
`any` implies that there might not be any, but if the file is rejected for being too big, I think there have to be some.